cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/YamlConfig.kt

```diff
@@ -43,6 +47,23 @@ class YamlConfig(
         val directoriesToPull: List<String> = listOf()) {
 
 
+    private val storage = StorageOptions.newBuilder().setProjectId(projectId).build().service
+    private val bucketLabel: Map<String, String> = mapOf(Pair("flank", "apks"))
```

Comment:
I think the value can be an empty string instead of `apks`. If flank is used on an iOS app then apks doesn't make sense.


> mapOf(Pair("flank", ""))